### PR TITLE
feat(docs): FRMW-162 - Migrate rxAttributes docs to ngdocs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-162

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxAttributes/rxAttributes.js [hunk 1]
```diff
@@ -4,7 +4,43 @@
...
+ * This component allows you to add attributes based on a value in scope being defined or not.
+ *
+ * ## Example
+ *
+ * <pre>
+ * <div rx-attributes="{'my-custom-attr': customAttrVal, 'ng-click': noFunc}"
+ *      ng-controller="myCtrl">
+ * </div>
+ * <script>
+ * function myCtrl (scope) {
```
Comment (CITguy) [inline src/rxAttributes/rxAttributes.js]: sadly, this is legacy controller syntax. Please update to reflect the following:

```javascript
angular.module('demoApp', [])
.controller('myCtrl', function ($scope) {
    $scope.customAttrVal = 'some value';
});
```

Comment (CITguy) [inline src/rxAttributes/rxAttributes.js]: feel free to separate the HTML and javascript into separate `<pre>` blocks

Comment (bry4n) [inline src/rxAttributes/rxAttributes.js]: updated! :+1:

File: src/rxAttributes/rxAttributes.js [hunk 2]
```diff
@@ -13,8 +51,12 @@ angular.module('encore.ui.rxAttributes', [])
 /**
  * @ngdoc directive
  * @name rxAttributes.directive:rxAttributes
- * @restrict E
- * @description [TBD]
+ * @restrict A
+ * @description
+ *
+ * This component allows you to add attributes based on a value in scope being defined or not.
```
Comment (CITguy) [inline src/rxAttributes/rxAttributes.js]: This description pertains to the `rxAttributes` directive, not the component.  Please replace `component` with `directive`.

Comment (bry4n) [inline src/rxAttributes/rxAttributes.js]: done!

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**